public/app/features/dashboard-scene/scene/ScopesScene.test.tsx

```diff
@@ -52,8 +52,8 @@ const scopesMocks: Record<
       description: 'Description 1',
       category: 'Category 1',
       filters: [
-        { key: 'a-key', operator: '=', value: 'a-value' },
-        { key: 'b-key', operator: '!=', value: 'b-value' },
+        { key: 'a-key', operator: 'equals', value: 'a-value' },
+        { key: 'b-key', operator: 'not-equals', value: 'b-value' },
```

Comment:
@dprokop Is it ok to do that?

Reply:
Correct me if I'm wrong @bfmatei , but I think it's fine. `operator` is simply a string, we have a [static list of operators for filters defined](https://github.com/grafana/scenes/blob/main/packages/scenes/src/variables/adhoc/AdHocFiltersVariable.tsx#L248) , but in the end it's for the data source to interpret the filter model.  

Eventually in the future it would be the data source - or rather the returned dimension type - that would indicate what operators are applicable for a given dimension, but that's still sth that we need to figure out. In this case I don't think the content of `operator` should matter.

I see @bergquist introduced `FilterOperator*` const in https://github.com/grafana/grafana/pull/84955 that you are relying on. But this creates a funky situation, where scope filters operator are data source agnostic, while `AdHocFilters` rely on prometheus-ish operators (see [this](https://github.com/grafana/scenes/blob/main/packages/scenes/src/variables/adhoc/AdHocFiltersVariable.tsx#L248)). It would probably be good if we aligned that between ad hoc filters and scope filters. But that's a work for a separate PR.

Reply:
So my point is - if you wanna use descriptive operators (`equals`, `non-equals` etc instead of `=`, `!=`), then you need to handle the mapping of the filters's provided operators to the descriptive ones supported by Prom data source.

Reply:
Yup, this is fine. The operators are not the same across all datasources so it's better to use descriptive + mappings instead of relying on Prom-like operators and map from those to whatever another datasource expects.